homeassistant/components/homee/entity.py

```diff
@@ -27,14 +27,20 @@ def __init__(self, attribute: HomeeAttribute, entry: HomeeConfigEntry) -> None:
         )
         self._entry = entry
         node = entry.runtime_data.get_node_by_id(attribute.node_id)
-        self._attr_device_info = DeviceInfo(
-            identifiers={
-                (DOMAIN, f"{entry.runtime_data.settings.uid}-{attribute.node_id}")
-            },
-            name=node.name,
-            model=get_name_for_enum(NodeProfile, node.profile),
-            via_device=(DOMAIN, entry.runtime_data.settings.uid),
-        )
+        ## Homee hub itself has node-id -1
```

Comment:
```suggestion
        # Homee hub itself has node-id -1
```
Didn't we create devices for these before?

Reply:
That was in the `HomeeNodeEntity` for the `HomeeNodeSensor`.
This is the first `HomeeEntity` the Homee Hub gets and thus I now have to create it here as well.